application/src/main/resources/security.yml

```diff
@@ -24,6 +24,9 @@ cql:
     keystore_password: ecchronos
     truststore: /path/to/truststore
     truststore_password: ecchronos
+    certificate:
```

Comment:
can we change these so they kinda match the ones for springboot? i.e:
certificate_private_key
trust_certificate

Reply:
Yes I will change.